src/assets/styles/header.scss

```diff
@@ -3,6 +3,15 @@
   text-align: center;
 }
 
+.header.-sticky {
```

Comment:
dang! i even tested this menu but must have not done the right combo of things. I will fix!